binary_class/doc/requirements.txt

```
@@ -1,4 +1,4 @@
 tensorflow
-bdrk[analyzer,model-monitoring]>=0.9,<0.10
+bdrk[analyzer]>=0.9.1,<0.10
```

```suggestion
bdrk[analyzer]>=0.9.1,<0.10
boxkite==0.0.5
```

do we need boxkite here?

---

not using model-monitoring, so no need for boxkite